lib/generators/pakyow/app/templates/app/setup.rb

```diff
@@ -13,6 +13,7 @@
 
     # put global config here and they'll be available across environments
     app.name = '<%= app_name %>'
+    logger.enabled = true
```

Comment:
If you think that would be the best approach I'd be happy to implement it that way, but it would leave the template generated by master broken by default.
I had another approach that I'd be curious to get your feedback on.  Checking Gem#latest_version_for checks remote repos to get the actual latest version of a Gem available and is therefore unusably slow.  What if instead we just check the latest version on the local machine where the generator is being executed and go with the assumptions that if the Pakyow gem doesn't exist on the local machine or if the version being used to generate the template is newer than the latest version installed on the local machine, than the person is probably working with the master repo and the Gemfile should therefore point to master.  In my mind then the only situation where the Gemfile would point to the wrong repo would be if someone cloned an old version (not master) of Pakyow via Git without installing the gem on their system which seems like an extreme edge case to me and would be worth the significant performance improvement gained.  I was thinking something like this:
```
local_pakyow = Gem::Specification.sort_by{ |g| [g.name.downcase, g.version] }.group_by{ |g| g.name }.detect{|k,v| k == 'pakyow'}
if local_pakyow && local_pakyow.last.last.version < Gem::Version.new(Pakyow::VERSION)
gem 'pakyow', github: 'pakyow/pakyow'
...
```
What do you think?